.rhcicd/clowdapp-backend.yaml

```diff
@@ -145,14 +145,10 @@ objects:
           value: ${OB_ENABLED}
         - name: OB_BRIDGE_UUID
           value: ${OB_BRIDGE_UUID}
-        - name: OB_KCUSER
-          value: ${OB_KCUSER}
-        - name: OB_KCPASS
-          value: ${OB_KCPASS}
-        - name: OB_TOKEN_USER
-          value: ${OB_TOKEN_USER}
-        - name: OB_TOKEN_PASS
-          value: ${OB_TOKEN_PASS}
+        - name: OB_TOKEN_CLIENT_ID
+          value: ${OB_TOKEN_CLIENT_ID}
+        - name: OB_TOKEN_CLIENT_SECRET
+          value: ${OB_TOKEN_CLIENT_SECRET}
```

Comment:
Gentle reminder (for later): this should be stored in the vault.

Reply:
Yes, I agree